register.php

```diff
@@ -6,7 +6,16 @@
     'leantime.core.template.tpl.timesheets.showAll.afterScriptLibTags',
     function (array $context) {
         if ('timesheets.showAll' === ($context['current_route'] ?? null)) {
-            echo '<script src="/dist/js/plugin-DataExport.js"></script>';
+            echo '<script src="/dist/js/plugin-AllTimesheetsDataExport.js"></script>';
```

Comment:
We should add some cache busting here, e.g.

```suggestion
            echo '<script src="/dist/js/plugin-AllTimesheetsDataExport.js?'.html_build_query(['v' => '%%VERSION%%']).'"></script>';
```

and then have the `create-release` script replace `%%VERSION%%` with the actual version.